wallstation_modules/revertbeheading/code/head.dm

```diff
@@ -0,0 +1,2 @@
+/obj/item/bodypart/head
+	can_dismember = TRUE
```

Comment:
```suggestion
// ORIGINAL FILE: code/modules/surgery/bodyparts/head.dm
/obj/item/bodypart/head
	can_dismember = TRUE
```

Is this repository supposed to contain some documentation for the modules? Like readme.md or just comments.

Reply:
I got no idea. For something this simple that isn't editing any TG files i think a comment is fine.